RELEASE.md

```diff
@@ -1,3 +1,16 @@
+# (unreleased)
+- Added an automatic fix for removable singularities in GHK equations. This feature has been moved from chaste_codegen to allow more general use.
```

Comment:
This looks like too much information for a changelog! Would suggest replacing this with e.g. "Added a method Model.fix_singularities()" and write this all up in the docstring there